### PR TITLE
Add TTBR information to Cloud documentation

### PR DESCRIPTION
**Pull request by sanderson**
Closes influxdata/DAR#302

- [x] Rebased/mergeable

### CODE REVIEW

**Review by philjb — CHANGES_REQUESTED** [review]
waiting on this until i get back from holiday

File: content/influxdb/cloud/reference/internals/ttbr.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+Whenever you send a write request to the `/api/v2/write` endpoint, the following occurs:
+
+- Validates the request and queues the write.
+- If the write is queued, responds with an HTTP 204 status code.
+- Handles the write asynchronously and reaches eventual consistency.
+
+_For more information, see [`/api/v2/write` documentation](/influxdb/cloud/api/#operation/PostWrite)._
+
+{{% note %}}
+The returned 204 status code code does not mean that the point is queryable;
```
Comment (btasker) [inline content/influxdb/cloud/reference/internals/ttbr.md:37]: This is a good thing to have in there, might even be worth cross-linking to https://docs.influxdata.com/influxdb/cloud/write-data/troubleshoot/#handle-write-and-delete-responses? That page gives quite a solid overview of the write process and why a write might fail even though it was a 204.

**Review by btasker — APPROVED** [review]
LGTM, thanks!

**Review by kelseiv — APPROVED** [review]

**Comment by sanderson** [comment]
Still getting pressure for this information from a few directions. I'm going to go ahead and merge this as a stop gap. I think the potential of publishing real-time TTBR numbers is a conversation that needs to happen between Product, Engineering, and SRE.

**Pull request merged**